app/locale/tr.coffee

```diff
@@ -1,102 +1,101 @@
+    for_teachers_subtitle3: "Yerleşik Değerlendirmeler"
+    for_teachers_subblurb3: "Öğretimi kişiselleştirin ve öğrencilerin biçimlendirici ve özetleyici değerlendirmelerle temel kavramları anlamalarını sağlayın."
+    for_teachers_subtitle4: "Otomatik Farklılaşma"
+    for_teachers_subblurb4: "Her öğrencinin öğrenme ihtiyaçlarına uyum sağlayan uygulama düzeyleriyle tüm öğrencilerin ilgisini farklı bir sınıfta çekin."
+    game_based_blurb: "CodeCombat, öğrencilerin gerçek kod yazdığı ve karakterlerinin gerçek zamanlı olarak tepki verdiğini gördüğü, oyun tabanlı bir bilgisayar bilimi programıdır."
     get_started: "Başlayalım"
-#    global_title: "Join Our Global Community of Learners and Educators"
-#    global_subtitle1: "Learners"
-#    global_subtitle2: "Lines of Code"
+    global_title: "Küresel Öğrenciler ve Eğitimciler Topluluğumuza Katılın"
+    global_subtitle1: "Öğrenciler"
+    global_subtitle2: "Kod Satırları"
     global_subtitle3: "Öğretmenler"
     global_subtitle4: "Ülkeler"
     go_to_my_classes: "Sınıfıma git"
     go_to_my_courses: "Kursuma git"
-#    quotes_quote1: "Name any program online, I’ve tried it. None of them match up to CodeCombat & Ozaria. Any teacher who wants their students to learn how to code... start here!"
-#    quotes_quote2: " I was surprised about how easy and intuitive CodeCombat makes learning computer science. The scores on the AP exam were much higher than I expected and I believe CodeCombat is the reason why this was the case."
-#    quotes_quote3: "CodeCombat has been the most beneficial for teaching my students real-life coding capabilities. My husband is a software engineer and he has tested out all of my programs. He put this as his top choice."
-#    quotes_quote4: "The feedback … has been so positive that we are structuring a computer science class around CodeCombat. The program really engages the students with a gaming style platform that is entertaining and instructional at the same time. Keep up the good work, CodeCombat!"
-#    quotes_quote5: "Even though the class starts every Saturday at 7am, my son is so excited that he wakes up before me! CodeCombat creates a pathway for my son to advance his coding skills."
-#    quotes_quote5_author: "Latthaphon Pohpon, Parent"
-#    see_example: "See example"
+    quotes_quote1: "Çevrimiçi herhangi bir program adlandırın, denedim. Hiçbiri CodeCombat ve Ozaria ile eşleşmiyor. Öğrencilerinin kodlamayı öğrenmesini isteyen herhangi bir öğretmen... buradan başlayın!"
+    quote_quote2: " CodeCombat'ın bilgisayar bilimi öğrenmeyi ne kadar kolay ve sezgisel hale getirdiğine şaşırdım. AP sınavındaki puanlar beklediğimden çok daha yüksekti ve bunun böyle olmasının nedeninin CodeCombat olduğuna inanıyorum."#    quotes_quote3: "CodeCombat has been the most beneficial for teaching my students real-life coding capabilities. My husband is a software engineer and he has tested out all of my programs. He put this as his top choice."
```

Comment:
u have joined quote2 and quote3, both should be on the separate lines like before